content/teknologi/altinnstudio/altinn-api/models/instance/_index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Instance
+description: An instance is a metadata container that is used to track the status and progress of one interaction between an App user and owner.
+toc: true
+tags: [api]
+weight: 40
+---
+
+## Instance
+
+The Instance model is the main model of an instance. An instance works as a form of envelope or folder where data can be collected and exchanged between the user and owner of the application. The instance document is a way for Altinn and external parties to track the state of one specific data exchange. 
```

Comment:
```suggestion
The Instance model is the main model of an instance. An instance can be regarded as an envelope or folder where data is collected and exchanged between the application user and owner.  The instance document is a way for Altinn and external parties to track the state of one specific data exchange. 
```

Reply:
Not sure if this is a bettwe way of formulating it, but I would like to avoid the term "a form of".

Reply:
I like it.